Minio/ApiEndpoints/BucketOperations.cs

```diff
@@ -88,11 +91,15 @@ public async Task MakeBucketAsync(string bucketName, string location = "us-east-
                 XmlSerializer = new RestSharp.Serializers.DotNetXmlSerializer(),
                 RequestFormat = DataFormat.Xml
             };
+
             // ``us-east-1`` is not a valid location constraint according to amazon, so we skip it.
             if (location != "us-east-1")
             {
                 CreateBucketConfiguration config = new CreateBucketConfiguration(location);
-                request.AddBody(config);
+                request.XmlSerializer.Namespace = "http://s3.amazonaws.com/doc/2006-03-01/";
+                request.XmlSerializer.ContentType = config.ToString();
+                string body = Regex.Replace(request.XmlSerializer.Serialize(config), @"\t|\n|\r", "");
```

Comment:
there is no need to use regex - just serialize XML correctly here and UTF8 encode content

Reply:
The changes to MakeBucketsAsync skipped my mind. My bad. Will change right away.